Remotion/Data/DomainObjects.UnitTests/Persistence/ReadOnlyStorageProviderManagerTest.cs

```diff
@@ -55,8 +55,10 @@ public void GetMandatory_WithProviderID ()
 #pragma warning restore CS0618 // Type or member is obsolete
 
       Assert.That(provider, Is.Not.Null);
-      Assert.That(provider.GetType(), Is.EqualTo(typeof(RdbmsProvider)));
-      Assert.That(provider.As<RdbmsProvider>().StorageProviderDefinition.Name, Is.EqualTo(c_testDomainProviderID));
+      Assert.That(provider.GetType(), Is.EqualTo(typeof(ReadOnlyStorageProviderDecorator)));
```

Comment:
I did not notice this before, but we should use 
Assert.That(provider, Is.InstanceOf<ReadOnlyStorageProviderDecorator>());
here and in line 70